test/CMakeLists.txt

```diff
@@ -14,3 +14,5 @@ foreach(test ${tests})
 endforeach()
 
 add_custom_target(test-verbose COMMAND ${CMAKE_CTEST_COMMAND} -C Debug --verbose)
+
+add_subdirectory(style)
```

Comment:
Okay, will remove this change for now

As a note for future. In my impression the style appears to be a crucial part of the CI test for CPM. Presently this feels odd that style is separate. Possibly it could be added always but include a new option CPM_INCLUDE_TEST_STYLE?

Reply:
DOne